pkg/controller/provider/web/base/client.go

```diff
@@ -13,6 +15,8 @@ import (
 	"reflect"
 )
 
+const ServiceCAFile = "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
```

Comment:
I don't think we want hard coded paths.
Recommend adding:
```
Inventory.TLS struct {
      ...
      // CA certificate path.
      CA string
}
```

Reply:
This is a hard coded path in OpenShift.